core/bones/base.py

```diff
@@ -914,4 +1041,12 @@ def structure(self) -> dict:
             }
         else:
             ret["multiple"] = self.multiple
+        if self.compute:
+            ret["compute"] = {
+                "method": self.compute.interval.method.name
+            }
+
+            if self.compute.interval.lifetime:
+                ret["compute"]["lifetime"] = str(self.compute.interval.lifetime)
```

Comment:
Shall we generally return seconds here? Currently, it's "0:00:30" for `timedelta(seconds=30)`.

Reply:
@ArneGudermann Why do we need to store this?